### DISCUSSION TITLE
Osu lazer not installing and/or opening

### DISCUSSION THREAD

**Discussion by sorencoke** [ppy/osu#16468] [Q&A] [answered]
I tried installing the newest 2022 release of osu lazer today, and it opens on the taskbar but shows nothing on screen, then installs a shortcut on my desktop. Clicking it does nothing. I tried adding --sdl to the shortcut target, from a suggestion in a similar thread but nothing happened. In event viewer I get 4 things  when opening osu! lazer, two say error two say information, I will link them. 

My specs are a Gtx 1060 6gb and a Amd ryzen 5 5600x. Let me know if you need more information
[event viewer osu logs.zip](https://github.com/ppy/osu/files/7875881/event.viewer.osu.logs.zip)
 
I also have tried using older versions and they still didn't work, maybe I can go further back tho

**Comment by bdach** [comment, marked as answer]
one of these shows:

```
Description: The process was terminated due to an unhandled exception.
Exception Info: Microsoft.Data.Sqlite.SqliteException (0x80004005): SQLite Error 26: 'file is not a database'.
   at Microsoft.Data.Sqlite.SqliteException.ThrowExceptionForRC(Int32 rc, sqlite3 db)
   at Microsoft.Data.Sqlite.SqliteCommand.PrepareAndEnumerateStatements(Stopwatch timer)+MoveNext()
   at Microsoft.Data.Sqlite.SqliteCommand.ExecuteReader(CommandBehavior behavior)
   at Microsoft.Data.Sqlite.SqliteCommand.ExecuteReader()
   at Microsoft.Data.Sqlite.SqliteCommand.ExecuteNonQuery()
   at osu.Game.Database.OsuDbContext..ctor(String connectionString)
   at osu.Game.Database.DatabaseContextFactory.CreateContext()
   at System.Threading.ThreadLocal`1.GetValueSlow()
   at System.Threading.ThreadLocal`1.get_Value()
```

please upload your `client.db` file. it should be one directory above the logs. after uploading it, rename it to something else temporarily and check if lazer opens correctly.

**Reply by sorencoke** [reply]
I don't see anything above the logs, am I looking in the right place?
![Desktop Screenshot 2022 01 15 - 13 10 06 39](https://user-images.githubusercontent.com/87891795/149637848-6daf59e4-2691-4102-837a-fe431b230a70.png)

**Reply by bdach** [reply]
right, forgot this was from event viewer. open file explorer and go to `%AppData%/osu` (paste that into the file path bar). `client.db` should be there.

**Reply by sorencoke** [reply]
Oh thanks, found it.
[client.zip](https://github.com/ppy/osu/files/7875898/client.zip)

**Reply by sorencoke** [reply]
I renamed it and it created a new client.db file and didn't open

**Reply by sorencoke** [reply]
I fixed it, I right clicked the shortcut on my desktop and saw that under start in it had a path that was a 2021 version of osu, not the one I downloaded, so I just deleted the appdata osu folder and reinstalled and its working.

**Reply by bdach** [reply]
your old database file appears to indeed have been corrupted in a manner that is not clear. i don't know why the game wouldn't open after deleting it though.

can you upload the recreated `client.db`? and also check event viewer again and see if any of the error messages are different?
